frontend/src/app/shared/components/blog-item-preview/blog-item-preview.component.scss

```diff
@@ -0,0 +1,70 @@
+@import 'variables';
+@import 'colors';
+
+
+.blog {
+  width: 470px;
+  height: 400px;
+
+  .gradient {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    padding: 24px;
+    border-radius: $border-radius-xl;
+    background: rgb(255,255,255);
+    background: linear-gradient(180deg, rgba(255,255,255,0) 0%, rgba(0,0,0,0) 50%, rgba(0,0,0,1) 100%);
+
+    .top {
+      width: 100%;
+      height: 50%;
+    }
+
+    .bottom {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      height: 50%;
+      color: $color-white;
+
+      .headline-text{
+        font-family: 'Poppins';
+        font-weight: 600;
+        font-size: 22px;
+        line-height: 33px;
+        padding: 40px 40px 8px 0px;
+      }
+
+      .under-text {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        width: 100%;
+
+
+        .avatar-image {
+          background-color: $color-white;
+          height: 32px;
+          width: 32px;
+          border-radius: 50%;
+          margin-right: 8px;
```

Comment:
ordoneaza-le https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
fixed